src/DonationForms/V2/resources/components/Onboarding/Dialogs/FeatureNoticeDialog.tsx

```diff
@@ -38,12 +47,13 @@ export default function FeatureNoticeDialog({isUpgrading, isEditing, handleClose
                     <StarsIcon /> {__("What's new", 'give')}
                 </div>
 
-                <p className={styles.message}>
-                    {__(
-                        'GiveWP 3.0 introduces an enhanced forms experience powered by the new Visual Donation Form Builder. The team is still working on add-on and gateway compatibility. If you need to use an add-on or gateway that isn\'t listed, use the "Add form" option for now.',
-                        'give'
-                    )}
-                </p>
+                <Interweave tagName={'p'} className={styles.message} content={
+                    sprintf(
+                        __('GiveWP 3.0 introduces an enhanced forms experience powered by the new Visual Donation Form Builder. The team is still working on add-on and gateway compatibility. If you need to use an add-on or gateway that isn\'t listed, use the "%sAdd form%s" option for now.', 'give'),
+                        '<strong>',
+                        '</strong>'
+                    )
+                } />
```

Comment:
Just in case you don't wanna import the Interweave library only for this particular usage, we can leverage on WP package instead. (Highly optional though)

```suggestion
                <p className={styles.message}>
                    {createInterpolateElement(
                        sprintf(__('GiveWP 3.0 introduces an enhanced forms experience powered by the new Visual Donation Form Builder. The team is still working on add-on and gateway compatibility. If you need to use an add-on or gateway that isn\'t listed, use the "%sAdd form%s" option for now.', 'give'), '<b>','</b>'),
                        {
                            b: <strong />,
                        }
                    )}
                </p>
```

Reply:
Awesome. Thanks for the suggestion!